core/src/main/java/org/pentaho/di/core/database/Database.java

```diff
@@ -2858,11 +2858,7 @@ public void setProcLookup( String proc, String[] arg, String[] argdir, int[] arg
   }
 
   public Object[] getLookup( boolean failOnMultipleResults ) throws KettleDatabaseException {
-    return getLookup( failOnMultipleResults, false );
-  }
-
-  public Object[] getLookup( boolean failOnMultipleResults, boolean lazyConversion ) throws KettleDatabaseException {
```

Comment:
Why was this method removed ? It's a public method in Database, so it could be called by lots of other things. Don't believe you can do this.